frontend/src/modules/frameworkImport/views/frameworkImportView.js

```diff
@@ -75,22 +75,23 @@ define(function(require){
       Origin.router.navigateToHome();
     },
 
-    onFormSubmitSuccess: function(importData, impoprtStatus, importXhr) {
+    onFormSubmitSuccess: function(data, importStatus, importXhr) {
       Origin.router.navigateToHome();
     },
 
-    onAjaxSuccess: function() {
-      this.goBack();
+    onAjaxError: function(data, status, error) {
+      var resJson = data.responseJSON || {};
+      var title = resJson.title || Origin.l10n.t('app.importerrortitle');
+      var msg = resJson.body && resJson.body.replace(/\n/g, "<br />") || error;
```

Comment:
should this be:
`var msg = (resJson.body && resJson.body.replace(/\n/g, "<br />")) || error;`